zarf.yaml

```diff
@@ -73,6 +73,13 @@ components:
               echo "This cluster can be destroyed with:"
               echo "k3d cluster delete ${ZARF_VAR_CLUSTER_NAME}"
             description: "Print out information about how to access the cluster remotely"
+          # Wait for CoreDNS to be available. This allows slow internet connections to complete before we move on and try to connect to a cluster that may or may not be ready to accept connections. CoreDNS is a deployment we can trust will always be there.
+          - wait:
+              cluster:
+                kind: deployment
+                condition: available
+                name: coredns
+                namespace: kube-system
```

Comment:
Might advocate for putting this either:
1. In a new `after` section in the actions: This would run before `onSuccess`, and in terms of semantics feels maybe more accurate since we don't know if the cluster setup was successful yet?
1. Before the other command in `onSuccess`: Would behave about the same timing wise.

One other comment - can we add a description to this (`description: "Wait for CoreDNS to be ready"` as a suggestion)?

Reply:
(also open to other opinions on this - this should be a 5 minute timeout now, which feels excessive if there were an issue with the deploy but maybe is helpful in case internet is _REALLY_ slow?)

Reply:
> This would run before onSuccess

`onSuccess:` isn't a child of before. It's a sibling. If we were to put it in `after:` then it would try to run even if the cluster deploy failed, which isn't necessary.

Reply:
Added a description

Reply:
Moved it to before the information print